content/en/monitors/configuration/_index.md

```diff
@@ -181,8 +181,8 @@ The selected behavior is applied when a monitor's query does not return any data
 
 The `Evaluate as zero` and `Show last known status` options are displayed based on the query type:
 
-- **Evaluate as zero:** This option is available for monitors using `Count` queries without the `default_zero()` function.
-- **Show last known status:** This option is available for monitors using any other query type than `Count`, for example `Gauge`, `Rate`, and `Distribution`, as well as for `Count` queries with `default_zero()`.
+- **Evaluate as zero:** For Metrics monitors, this option is available for queries where each metric either uses the `sum` space aggregation, is a count of distribution values, or uses one of these functions: `default_zero()`, `count_nonzero()` or `count_not_null()`. For all other monitors, this option is available for monitors using `Count` queries without the `default_zero()` function.
+- **Show last known status:** This option is available for monitors using any other query aggregations or formulas that could divide by zero.
```

Comment:
**Evaluate as zero:** 
: This option is available under specific conditions for Metrics monitors and other types of monitors.<br><br> For Metrics monitors, this option is available when either of the following occurs:<br>- Each metric uses the `sum` space aggregation.<br> - The metric is a count of distribution values.<br>  - The metric uses one of the following functions: `default_zero()`, `count_nonzero()`, or `count_not_null()`.<br><br> For other monitors, the option is applicable when:<br>- The monitor uses `Count` queries without the `default_zero()` function.

**Show last known status:** 
: This option is available for monitors using any other query aggregations or formulas that could divide by zero.

Reply:
Optional formatting